app/src/main/java/org/mifos/selfserviceapp/ui/adapters/SavingAccountsListAdapter.java

```diff
@@ -54,6 +59,29 @@ public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
             ((ViewHolder) holder).tv_savingAccountProductName.setText(
                     savingAccount.getProductName());
 
+
```

Comment:
I have looked the adapter and found ``` private final LayoutInflater layoutInflater``` is unused remove this and 
```java
((ViewHolder) holder).tv_clientSavingAccountNumber.setText(
                    savingAccount.getAccountNo().toString());
``` here Account No already a string, So toString() is redundant , remove toString().
and 
``` if (holder instanceof RecyclerView.ViewHolder)``` this check is redundant because
```RecyclerView.ViewHolder``` is RecyclerView abstract standard class but here we have check holder is the instance of the SavingAccountsListAdapter.ViewHolder or not, So remove ```RecyclerView.ViewHolder``` to SavingAccountsListAdapter's ViewHolder.